### PR TITLE
Update Project.toml

### PR DESCRIPTION
**Pull request by mforets**
See #1781.

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -19,26 +19,26 @@ SharedArrays = "1a1011a3-84de-559e-8e89-a11a2f7dc383"
...
-Expokit = "≥ 0.1.0"
-GLPK = "≥ 0.10.0"
-GLPKMathProgInterface = "≥ 0.4.0"
-GR = "≥ 0.39.1"
-IntervalArithmetic = "≥ 0.14.0"
-IntervalMatrices = "≥ 0.2.1"
+Expokit = "0.2.0"
+GLPK = "0.11.4"
+GLPKMathProgInterface = "0.4.4"
+GR = "0.42.0"
```
Comment (mforets) [inline Project.toml]:
```suggestion
GR = "0.41.0"
```

File: Project.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name = "LazySets"
 uuid = "b4f0291d-fe17-52bc-9479-3d1a343d9043"
-version = "1.21.0"
+version = "1.22.0"
```
Comment (schillic) [inline Project.toml]: Can we do this in a separate PR please?

File: Project.toml [hunk 3]
```diff
@@ -21,24 +21,24 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
...
-Makie = "≥ 0.9.4"
-MathProgBase = "≥ 0.7.0"
-Optim = "≥ 0.14.1"
-Plots = "≥ 0.25.1"
-Polyhedra = "0.5.4"
-RecipesBase = "≥ 0.3.0"
-Reexport = "≥ 0.2.0"
-Requires = "≥ 0.4.0"
-TaylorModels = "≥ 0.1.1"
+Expokit = "^0.2"
```
Comment (schillic) [inline Project.toml]: Any reason why `Expokit` v0.1 is not allowed anymore? The old version allowed it and it seemed to work.
The same holds for `GR`, `IntervalArithmetic`, `RecipesBase`, `Requires`, and `TaylorModels`.
If there is a reason to upgrade one of those packages, I think we should add a test to capture this necessity.

File: Project.toml (hunk 2 above)
Comment (mforets) [inline Project.toml]:
```suggestion
version = "1.21.0"
```

File: Project.toml (hunk 3 above)
Comment (mforets) [reply Project.toml]: > Any reason why Expokit v0.1 is not allowed anymore? The old version allowed it and it seemed to work.

See: This release establishes compatibility with Julia 0.7/1.0. From now on support for Julia 0.6 will be dropped, https://github.com/acroy/Expokit.jl/releases/tag/v0.2.0

> The same holds for GR, IntervalArithmetic, RecipesBase, Requires, and TaylorModels.
If there is a reason to upgrade one of those packages, I think we should add a test to capture this necessity.

I don't think that we need to add more tests for backwards compatibility  -- if it is not broken, don't fix it :) .

File: Project.toml [hunk 4]
```diff
@@ -21,24 +21,24 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
...
+GLPK = "^0.10, ^0.11"
+GLPKMathProgInterface = "^0.4"
+GR = "^0.41"
```
Comment (schillic) [inline Project.toml]:
```suggestion
GR = "^0"
```
Your version is already outdated. Since this package is completely optional and I do not expect any breaking changes before v1 because we only use it through `Plots`, I say we do not need to add a bound for the minor version.

File: Project.toml (hunk 3 above)
Comment (schillic) [reply Project.toml]: Okay, I checked these packages and I agree to upgrade them.
